ERCS/erc-6596.md

```diff
@@ -13,485 +13,266 @@ requires: 721, 1155
-China Morning Post (SCMP) of the Sino-British Joint Declaration via the HAT metadata. While well-informed collectors may
-not need this programmed connectivity, a standardised metadata for HATs will make these connections easily accessible
-for all collectors, scholars, and interested parties.
-
-Provenance and context are extremely important for historical assets. The archival and research of significant artworks,
-objects, and collections are ongoing; it is a collaborative effort between creators, artist estates, galleries, auction
-houses, scholars and institutions. The consolidation of information and knowledge is not only crucial to the
-understanding of our cultural heritage; the richer the context, the more valuable the asset is to collectors. A shared
-and standardised metadata structure enriches the context of an NFT. With the establishment of HATs, it is our hope that
-this standardised structure will contribute to the evolution of cultural heritage, and fully capture the context of
-culturally and historically significant assets, thereby ensuring relevant and fair value for all historically
-significant NFTs.
+**Preserving context and significance** - Provenance and context are crucial for cultural and historical assets. The CHAT
+standard captures and preserves the provenance and history of these assets, as well as the changing contexts that emerge
+from new knowledge and information. This context and provenance substantiate the significance and value of cultural and
+historical assets.
+
+**Proof-based preservation** - The recent incident of lost artifacts at the British Museum[^1] points to a need in
+reassessing our current record keeping mechanisms. While existing systems mostly operate on trust, blockchain technology
+offers opportunities to establish permanent and verifiable records in a proof-based environment. Introducing the CHAT
+standard on the Ethereum platform enables museums, institutions, and owners of significant collections to create
+tamper-proof records on the blockchain. By representing these valuable cultural and historical assets as tokens on the
+blockchain, permanent and tamper-proof records can be established whenever amendments are made, ensuring greater
+transparency and accountability.
+
+**Interoperability** - The proposed standard addresses the multitude of existing metadata standards used in the arts and
+cultural sector, including CDWA, CDWA-Lite, CCO, CIDOC CRM, LiDO, ISAD(G), DACS, EAD, CCDO, Getty vocabularies (AAT &
+TGN), Europeana, SPECTRUM, CHIN, RAD, Dublin Core Metadata Standards, and VRA Core 4.0. The vision is to create a
+metadata structure specifically built for preservation on the blockchain that is interoperable with these existing
+standards and compliant with the Open Archives Initiative (OAI) as well as the International Image Interoperability
+Framework protocol (IIIF).
```

Comment:
That's a lot of alphabet soup. Generally we prefer expanding abbreviations the first time they are used. In this particular case, I'm not sure how much value there is in listing all these organizations/standards individually. Maybe just a blanket:

```suggestion
**Interoperability** - The proposed standard addresses the multitude of existing metadata standards used in the arts and
cultural sector. The vision is to create a
metadata structure specifically built for preservation on the blockchain that is interoperable with these existing
standards and compliant with the Open Archives Initiative (OAI) as well as the International Image Interoperability
Framework protocol (IIIF).
```

Reply:
Good point! Co-author updated as per suggestion.